ur_controllers/src/speed_scaling_state_controller.cpp

```diff
@@ -19,10 +19,187 @@
 //----------------------------------------------------------------------
 /*!\file
  *
- * \author  Felix Exner exner@fzi.de
- * \date    2019-04-17
+ * \author  Marvin Große Besselmann grosse@fzi.de
+ * \date    2021-02-10
  *
  */
 //----------------------------------------------------------------------
 
 #include "ur_controllers/speed_scaling_state_controller.h"
+
+#include <stddef.h>
+#include <limits>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "hardware_interface/types/hardware_interface_return_values.hpp"
+#include "hardware_interface/types/hardware_interface_type_values.hpp"
+#include "rclcpp/clock.hpp"
+#include "rclcpp/qos.hpp"
+#include "rclcpp/qos_event.hpp"
+#include "rclcpp/time.hpp"
+#include "rclcpp_lifecycle/lifecycle_node.hpp"
+#include "rcpputils/split.hpp"
+#include "rcutils/logging_macros.h"
+#include "std_msgs/msg/header.hpp"
+
+namespace rclcpp_lifecycle
+{
+class State;
+}  // namespace rclcpp_lifecycle
+
+namespace ur_controllers
+{
+const auto kUninitializedValue = std::numeric_limits<double>::quiet_NaN();
```

Comment:
Make CI happy: K_UNINITIALIZED_VALUE

Reply:
Value removed, since it was no longer necessary